packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -20,26 +24,70 @@ class PropertiesView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ValueListenableBuilder<List<RemoteDiagnosticsNode>>(
-      valueListenable: controller.selectedNodeProperties,
-      builder: (context, properties, _) {
-        return Container(
-          margin: const EdgeInsets.all(denseSpacing),
-          child: ListView.builder(
-            itemCount: properties.length,
-            itemBuilder: (context, index) {
-              return PropertyItem(
-                index: index,
-                properties: properties,
-              );
-            },
+    return MultiValueListenableBuilder(
+      listenables: [
+        controller.selectedNodeWidgetProperties,
+        controller.selectedNodeRenderProperties,
+      ],
+      builder: (context, values, _) {
+        final widgetProperties = values.first as List<RemoteDiagnosticsNode>;
+        final renderProperties = values.second as List<RemoteDiagnosticsNode>;
+        return Padding(
+          padding: const EdgeInsets.all(denseSpacing),
+          child: AnalyticsTabbedView(
```

Comment:
fine to land like this for now and we can iterate later. Another option we could consider instead of a tab view would be to store the render object properties in a collapsible list item / drawer beneath the widget properties